shell/platform/android/test/io/flutter/embedding/engine/systemchannels/SettingsChannelTest.java

```diff
@@ -23,6 +23,8 @@ public class SettingsChannelTest {
   @Test
   @TargetApi(33)
   @Config(sdk = 33)
+  @SuppressWarnings("deprecation")
+  // DartExecutor.send is deprecated.
```

Comment:
Can we reference an issue for all of these deprecated send calls (here and in other files)?